src/data-root.ts

```diff
@@ -0,0 +1,46 @@
+import localforage from 'localforage'
+
+import * as core from './core'
+import * as dns from './dns'
+import { CID } from './ipfs'
+import { UCAN_STORAGE_KEY } from './common'
+
+
+/**
+ * Get the CID of a user's data root.
+ */
+export async function dataRoot(username: string): Promise<CID> {
+  try {
+    // TODO: This'll be `files.${username}.fission.name` later
+    return await dns.lookupDnsLink(`${username}.fission.name`)
```

Comment:
such a small nitpick, but: you don't need the `await` here

Reply:
I'm using `async` to force it into throwing an error. Could have used `.catch()` alternatively, but felt like doing this 🤷

Reply:
ah didn't realize. good looks :+1: